controller_manager:
  ros__parameters:
    update_rate: 100
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
    forward_position_controller:
      type: forward_command_controller/ForwardCommandController
      joints:
        - Revolute_9
        - Revolute_10
        - Revolute_11
        - Revolute_12
        - Revolute_13
        - Slider_14
        - Slider_15
      state_publish_rate: 50
      constraints:
        goal_time: 0.2
        stopped_velocity_tolerance: 0.01
        Revolute_9:
          trajectory: 0.1
          goal: 0.1
        Revolute_10:
          trajectory: 0.1
          goal: 0.1
        Revolute_11:
          trajectory: 0.1
          goal: 0.1
        Revolute_12:
          trajectory: 0.1
          goal: 0.1
        Revolute_13:
          trajectory: 0.1
          goal: 0.1
        Slider_14:
          trajectory: 0.1
          goal: 0.1
        Slider_15:
          trajectory: 0.1
          goal: 0.1
      gains:
        Revolute_9:
          p: 100.0
          i: 0.1
          d: 0.01
        Revolute_10:
          p: 100.0
          i: 0.1
          d: 0.01
        Revolute_11:
          p: 100.0
          i: 0.1
          d: 0.01
        Revolute_12:
          p: 100.0
          i: 0.1
          d: 0.01
        Revolute_13:
          p: 100.0
          i: 0.1
          d: 0.01
        Slider_14:
          p: 100.0
          i: 0.1
          d: 0.01
        Slider_15:
          p: 100.0
          i: 0.1
          d: 0.01
